ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/filestore-csi-defaultsc/cucushift-installer-rehearse-gcp-ipi-filestore-csi-defaultsc-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: cucushift-installer-rehearse-gcp-ipi-filestore-csi-defaultsc
```

Comment:
I wonder the difference between `cucushift-installer-rehearse-gcp-ipi-filestore-csi` and `cucushift-installer-rehearse-gcp-ipi-filestore-csi-defaultsc`.  `cucushift-installer-rehearse-gcp-ipi-filestore-csi` just prepared filestore-csi storage, but never enabled it as default sc, if that, does that mean the storage is never used? If I am right, I think we should directly enable `filestore-csi` as default sc directly on `cucushift-installer-rehearse-gcp-ipi-filestore-csi` install workflow.

Reply:
Yes, `cucushift-installer-rehearse-gcp-ipi-filestore-csi` just prepared filestore storage and only storage team will run related test cases. Other team which use default sc will not use filestore volume.

Reply:
Sure.